ci_scripts/build_macos.sh

```diff
@@ -22,14 +22,19 @@ echo "Configure KTX-Software (macOS)"
 if [ -n "$MACOS_CERTIFICATES_P12" ]; then
   cmake -GXcode -Bbuild-macos \
   -DKTX_FEATURE_DOC=ON \
+  -DCMAKE_OSX_ARCHITECTURES="\$(ARCHS_STANDARD)" \
+  -DBASISU_SUPPORT_SSE=OFF \
   -DKTX_FEATURE_LOADTEST_APPS=ON -DVULKAN_INSTALL_DIR="${VULKAN_INSTALL_DIR}" \
   -DXCODE_CODE_SIGN_IDENTITY="${CODE_SIGN_IDENTITY}" \
   -DXCODE_DEVELOPMENT_TEAM="${DEVELOPMENT_TEAM}" \
   -DPRODUCTBUILD_IDENTITY_NAME="${PKG_SIGN_IDENTITY}"
 else # No secure variables means a PR or fork build.
   cmake -GXcode -Bbuild-macos \
   -DKTX_FEATURE_DOC=ON \
-  -DKTX_FEATURE_LOADTEST_APPS=ON -DVULKAN_INSTALL_DIR="${VULKAN_INSTALL_DIR}"
```

Comment:
Why is `-DVULKAN_INSTALL_DIR="${VULKAN_INSTALL_DIR}` removed from PR & fork builds but not from a origin branch builds?

B.t.w PR #427 will remove `VULKAN_INSTALL_DIR` from the build.

Reply:
It is not removed, but I split up those two in separate lines for readability. (see lines 36 and 37)

Thanks for pointing out `VULKAN_INSTALL_DIR` is removed.